src/pages/recipes.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { Divider } from '@chakra-ui/react';
+import { graphql, PageProps } from 'gatsby';
+import { Helmet } from 'react-helmet';
+import Layout from 'src/components/Layout';
+
+import styles from '../css/Blog.module.css';
+
+interface Props extends PageProps {
+  data: GatsbyTypes.RecipeQueryQuery;
+}
+
+function RecipesIndex(props: Props): JSX.Element {
+  const siteTitle = props.data.site?.siteMetadata?.title;
+  const posts = props.data?.allContentfulRecipe?.edges;
```

Comment:
nit: would prefer `recipes` rather than `posts`

Reply:
gotcha, will change it for future